tests/api/v1/generators/post.js

```diff
@@ -295,9 +295,9 @@ describe('tests/api/v1/generators/post.js >', () => {
       })
       .then((asp2) => { // created aspect2
         aspects.push(asp2.name);
-        asp2.addWriter(user2); // assign user2 as writer to aspect2
-        return validateGeneratorAspectsPermissions(aspects, req);
+        return asp2.addWriter(user2); // assign user2 as writer to aspect2
       })
+      .then(() => validateGeneratorAspectsPermissions(aspects, req))
```

Comment:
Good catch. Should it be the same on line 291?

Reply:
changed a few more, yeah

Reply:
tests passing